src/maintenance/azext_maintenance/azext_metadata.json

```diff
@@ -1,3 +1,4 @@
 {
-    "azext.minCliCoreVersion": "2.0.47"
+    "azext.isExperimental": true,
```

Comment:
Is it expected that all commands labelled as "Experimental" as before it's GA?

Reply:
Use "azext.isPreview": true here

---
In reply to: [478224362](https://github.com/Azure/azure-cli-extensions/pull/2214#discussion_r478224362) [](ancestors = 478224362)